pkgs/by-name/ro/rogauracore/package.nix

```
@@ -0,0 +1,53 @@
+{ lib, stdenv, fetchFromGitHub, libusb1, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "rogauracore";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    url = "https://github.com/wroberts/rogauracore";
+    rev = "a872431a59e47c1ab0b2a523e413723bdcd93a6e";
+    hash = "sha256-SeG6B9ksWH4/UjLq5yPncVMTYjqMOxOh2R3N0q29fQ0=";
+  };
```

Oh boy...

```suggestion
  src = fetchFromGitHub {
    owner = "wroberts";
    repo = "rogauracore";
    rev = version;
    hash = "";
  };
```

And recalculate the hash.